.github/workflows/release.yml

```
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  lint:
+    if: github.ref == 'refs/heads/main'
```

Each job in this workflow shall be triggered uniquely by https://github.com/bot-anik 😉 

```suggestion
    if: github.ref == 'refs/heads/main' && github.actor == 'bot-anik'
```